Auth0Tests/WebAuthSpec.swift

```diff
@@ -267,6 +267,10 @@ class WebAuthSpec: QuickSpec {
                 expect(newWebAuth().useUniversalLink().redirectURL?.absoluteString) == "https://\(Domain)/ios/\(bundleId)/callback"
             }
 
+            it("should build with a custom url") {
+                expect(newWebAuth().redirectURL(RedirectURL).redirectURL) == RedirectURL
```

Comment:
where is this value defined, should we hardcode here something different instead?

Reply:
That value is defined at the top of that file, and is used through the tests in said file.